src/Take.Blip.Builder.UnitTests/Actions/ProcessHttpActionTests.cs

```diff
@@ -177,7 +180,7 @@ public async Task ProcessActionWithBotIdentifierHeaderShouldSucceed()
 
             var settings = new ProcessHttpSettings
             {
-                Uri = new Uri("https://blip.ai"),
+                Uri = new Uri("http.msging.net"),
```

Comment:
this is no valid URI. Replace with "https://test.msging.net" for example.

Reply:
DOne